pulse_actions/handlers/config.py

```diff
@@ -10,6 +11,9 @@
     "exchange/treeherder/v1/resultset-actions": {
         "resultset_actions": treeherder_resultset.on_resultset_action_event
     },
+    "exchange/treeherder/v1/resultset-runnable-job-actions": {
```

Comment:
I'm going to work on this on a follow-up PR, config.py should be a dictionary of topic : function, instead of a dictionary of exchange : function. It is very confuse right now.